### ISSUE TITLE
Spyglass doesn't show buildlog

### ISSUE THREAD

**Issue by alvaroaleman** [kubernetes/test-infra#18968]
**What happened**:

Spyglass doesn't show the buildlog at all after upgrading from  43094a7133 to 64613f6ea7

/kind bug
/area prow
/area prow/deck

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Please provide links to example occurrences, if any**:

**Anything else we need to know?**:

**Comment by stevekuznetsov** [comment]
```
{
  "component": "deck",
  "container_name": "deck",
  "error": "template: template.html:10:9: executing \"body\" at <.ShowRawLog>: can't evaluate field ShowRawLog in type buildlog.LogArtifactView",
  "file": "prow/spyglass/lenses/buildlog/lens.go:351",
  "func": "k8s.io/test-infra/prow/spyglass/lenses/buildlog.executeTemplate",
  "host": "ip-10-0-133-202",
  "level": "error",
  "msg": "Error executing template.",
  "multiline_tag": "F",
  "object_uid": "7f6e1dec-9e70-4666-9604-47edb07697ee",
  "pod_name": "deck-6cbd8c9f94-2vtst",
  "pod_namespace": "ci",
  "severity": "error",
  "source_type": "kubernetes",
  "stream": "stderr",
  "time": "2020-08-24T21:10:48Z"
}
```

Culprit is https://github.com/kubernetes/test-infra/pull/18823

**Comment by alvaroaleman** [comment]
Fixed in #18979
